protos/protos.d.ts

```diff
@@ -0,0 +1,15286 @@
+import * as $protobuf from "protobufjs";
+/** Namespace google. */
+export namespace google {
+
+    /** Namespace protobuf. */
+    namespace protobuf {
+
+        /** Properties of a Duration. */
+        interface IDuration {
+
+            /** Duration seconds */
+            seconds?: (number|Long|null);
```

Comment:
@alexander-fenster nice to see these being generated now! Though it looks like the import for `Long` is missing